pints/residuals_diagnostics.py

```diff
@@ -0,0 +1,272 @@
+#
+# Functions for analyzing the residuals and evaluating noise models
+#
+# This file is part of PINTS
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing informating, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+import math
+
+
+def plot_residuals_autocorrelation(parameters,
+                                   problem,
+                                   max_lag=10,
+                                   thinning=None,
+                                   significance_level=0.05,
+                                   posterior_interval=0.95):
+    r"""
+    Generate an autocorrelation plot of the residuals.
+
+    This function can be used to analyze the results of either optimisation or
```

Comment:
analyse

It's no big deal but we'll need to be consistent

Reply:
Wasn't this fixed in b9fd6b4c76ac04d89f3adfcea79f56fbec573446